Kudu.Agent/Host.cs

```diff
@@ -0,0 +1,70 @@
+// ------------------------------------------------------------------------------
+//  <copyright file="Host.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+//  </copyright>
+// ------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Kudu.Agent
+{
+    public static class Host
+    {
+        public static void Run(string[] args)
+        {
+            AppDomain.CurrentDomain.UnhandledException += OnUnhandledException;
+
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder
+                        .ConfigureServices(services =>
+                        {
+                            services.AddControllers();
+                        })
+                        .UseKestrel(options =>
+                        {
+                            options.ListenAnyIP(port: 50555);
```

Comment:
@jvano Is it ok that we always use 50555? Or should it grab a port at random and set an environment variable equal to the port that it grabbed? I think this would cause an issue if the port was somehow already in use. Shouldn't happen bc this will start on startup of the container, but it still might be a good thing to avoid.